source/customization.rst

```diff
@@ -850,3 +850,59 @@ To resolve this set ``OOD_JOB_NAME_ILLEGAL_CHARS`` to ``/`` for all OOD applicat
   # /etc/ood/config/nginx_stage.yml
   pun_custom_env:
     - OOD_JOB_NAME_ILLEGAL_CHARS: "/"
+
+.. _customize_dex_theme:
+
+Customize Dex Theme
+-------------------
+
+It's possible to use a customized theme when authenticating with Dex when using OnDemand's default authentication.
+Refer to the upstream `Dex template docs`_ for additional information on templating Dex.
+
+The simplest approach is to copy the OnDemand theme and make changes.  This is idea if you wish to make the following changes:
+
+- Change navigation or login page logos
+- Change favicon
+- Change CSS styles
+
+.. code-block:: sh
+
+   cp -r /usr/share/ondemand-dex/web/themes/ondemand /usr/share/ondemand-dex/web/themes/mycenter
+
+To update the theme you must modify ``/etc/ood/config/ood_portal.yml`` and regenerate the Dex configuration:
+
+.. code-block:: yaml
+   :emphasize-lines: 3-
+
+   dex:
+   # ...
+     frontend:
+       theme: mycenter
+
+The default ``ondemand`` theme can also be configured using the following configuration keys within ``/etc/ood/config/ood_portal.yml``:
+
+.. code-block:: yaml
+   :emphasize-lines: 4-
+
+   dex:
+   # ...
+     frontend:
+       issuer: "MyCenter OnDemand"
+       extra:
```

Comment:
@msquee Does this example reflect correctly all the options you added to the templates for ondemand-dex?